### PR TITLE
Govcy file upload component

### PR DESCRIPTION
**Pull request by UI-TEC**

### CODE REVIEW

File: build/index.html [hunk 1]
```diff
@@ -937,6 +937,26 @@ <h2>Date of birth</h2>
    <br>
    <br>
 
+   <h2>File upload</h2>
+    <div class="govcy-container">
+        <form action="" class="govcy-form">
+            <fieldset class="govcy-fieldset">
```
Comment (cevangelougovcy) [inline build/index.html:943]: The input should not be in a `<fieldset>`.

**Review by cevangelougovcy — COMMENTED** [review]
Looks great, please check my comment

File: build/index.html (hunk 1 above)
Comment (UI-TEC) [reply build/index.html]: <fieldset> removed as per comment.

**Review by cevangelougovcy — APPROVED** [review]
Excellent thank you

**Pull request closed without merging**